marlowe-runtime/marlowe-tx/Main.hs

```diff
@@ -214,6 +219,14 @@ getOptions = execParser $ info (helper <*> parser) infoMod
       , showDefault
       ]
 
+    analysisTimeoutParser = option (fromInteger <$> auto) $ mconcat
+      [ long "analysis-timeout"
+      , value 15
```

Comment:
The slow phase of the safety analysis is limited to 15 seconds by default, at the `marlowe-tx` level.